### PR TITLE
Python SDK: updates to whats new

### PR DESCRIPTION
**Pull request by isvargasmsft**
Adding information about the Python SDK release to whats new topic. 

Blog post here: https://devblogs.microsoft.com/microsoft365dev/introducing-the-microsoft-graph-python-sdk-now-available-for-public-preview/

### CODE REVIEW

File: concepts/whats-new-overview.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ See highlights of what's new in the recent two months in Microsoft Graph, [what'
 ### People and workplace intelligence | Item insights
 A user whose item insights have been disabled can still see the file-based activities of other users with item insights enabled. Previous to this update, that user with item insights disabled would not see anybody's trending content. Learn more about [an organization's insights-based experience after disabling a user's item insights](item-insights-overview.md#disable-or-re-enable-item-insights).
 
+### Use SDKs
+Try the new [Microsoft Graph Python SDK (preview)](https://devblogs.microsoft.com/microsoft365dev/introducing-the-microsoft-graph-python-sdk-now-available-for-public-preview/) and take advantage of the following improvements:
+- A new authentication provider that automatically refreshes access tokens.
+- A built-in retry handler that understands response status codes.
+- A fluent request building pattern to improve efficiency and discoverability.
+- Type annotations, both sync and async experiences and HTTP2 support.
```
Comment (angelgolfer-ms) [inline concepts/whats-new-overview.md:25]: Line 25, please use synchronous and asynchronous.

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
The language use looks good. Sorry I'm on my way out to pick up my mom. Please make sure the content side reflects what customers need to know as top takeaway.

File: concepts/whats-new-overview.md [hunk 2]
```diff
@@ -17,6 +17,19 @@ See highlights of what's new in the recent two months in Microsoft Graph, [what'
...
+- A new authentication provider that automatically refreshes access tokens.
+- A built-in retry handler that understands response status codes.
+- A fluent request building pattern to improve efficiency and discoverability.
+- Type annotations, both synchronous and asynchronous experiences and HTTP2 support.
+
+To get started, see:
+- [README](https://github.com/microsoftgraph/msgraph-sdk-python#readme)
+- [Upgrade guide](https://github.com/microsoftgraph/msgraph-sdk-python/blob/main/UPGRADING.md)
+
+> **Note:** The Microsoft Graph Python SDK is currently in public preview. Don't use this SDK in production environments. For details see [SDKs in preview or GA status](https://learn.microsoft.com/en-us/graph/sdks/sdks-overview#sdks-in-preview-or-ga-status).
```
Comment (angelgolfer-ms) [inline concepts/whats-new-overview.md:31]: Line 31:
Let's not use the '>' styling for the note. The reason is the what's new page contains many new items for other workloads, it's better to maintain a more neutral purview over all the what's new items, not calling one out more than others.

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
One comment, please take a look.

File: concepts/whats-new-overview.md [hunk 3]
```diff
@@ -17,6 +17,19 @@ See highlights of what's new in the recent two months in Microsoft Graph, [what'
...
 
+### Use SDKs
+Try the new [Microsoft Graph Python SDK (preview)](https://devblogs.microsoft.com/microsoft365dev/introducing-the-microsoft-graph-python-sdk-now-available-for-public-preview/) and take advantage of the following improvements:
+- A new authentication provider that automatically refreshes access tokens.
+- A built-in retry handler that understands response status codes.
+- A fluent request building pattern to improve efficiency and discoverability.
+- Type annotations, both synchronous and asynchronous experiences and HTTP2 support.
+
+**Note**: The Microsoft Graph Python SDK is currently in public preview. Don't use this SDK in production environments. For details see [SDKs in preview or GA status](https://learn.microsoft.com/en-us/graph/sdks/sdks-overview#sdks-in-preview-or-ga-status).
```
Comment (isvargasmsft) [inline concepts/whats-new-overview.md:27]:
```suggestion
**Note**: The Microsoft Graph Python SDK is currently in public preview. Don't use this SDK in production environments. For details see [SDKs in preview or GA status](/graph/sdks/sdks-overview#sdks-in-preview-or-ga-status).
```

**Review by angelgolfer-ms — APPROVED** [review]
Your last fixes for the warnings look good.

File: concepts/whats-new-overview.md [hunk 4]
```diff
@@ -17,6 +17,19 @@ See highlights of what's new in the recent two months in Microsoft Graph, [what'
...
 A user whose item insights have been disabled can still see the file-based activities of other users with item insights enabled. Previous to this update, that user with item insights disabled would not see anybody's trending content. Learn more about [an organization's insights-based experience after disabling a user's item insights](item-insights-overview.md#disable-or-re-enable-item-insights).
 
+### Use SDKs
+Try the new [Microsoft Graph Python SDK (preview)](https://devblogs.microsoft.com/microsoft365dev/introducing-the-microsoft-graph-python-sdk-now-available-for-public-preview/) and take advantage of the following improvements:
+- A new authentication provider that automatically refreshes access tokens.
+- A built-in retry handler that understands response status codes.
+- A fluent request building pattern to improve efficiency and discoverability.
+- Type annotations, both synchronous and asynchronous experiences and HTTP2 support.
+
+**Note**: The Microsoft Graph Python SDK is currently in public preview. Don't use this SDK in production environments. For details see [SDKs in preview or GA status](/graph/sdks/sdks-overview#sdks-in-preview-or-ga-status).
```
Comment (angelgolfer-ms) [inline concepts/whats-new-overview.md:27]: Line 27:
FYI I called this out by just bolding "Note" to decrease the distraction from the rest of the non-Python-SDK items on the rest of the page. I also moved the note to this location for 2 reasons: 
- To make sure customers are aware if they're only interested in tools (SDK) they can use in a production app, then they don't have to proceed with the "get started" part. The flip side is true too, if they don't mind trying this out only in non-production apps, then they can follow the call to action ("get started").
- To leave the call to action ("get started") to the last with respect to this item, in this case. Usually we do that as the last thing, as the customer may leave this topic at this point.

**Pull request merged**